packages/connection-storage/src/connection-storage.ts

```diff
@@ -1,520 +1,170 @@
-          .filter((x) => x.connectionInfo?.connectionOptions?.connectionString)
-          .map((connection) =>
-            this.mapStoredConnectionToConnectionInfo(connection)
-          )
-      );
-    } catch (err) {
-      log.error(
-        mongoLogId(1_001_000_101),
-        'Connection Storage',
-        'Failed to load connections',
-        { message: (err as Error).message }
-      );
-      return [];
-    }
+  }): Promise<ConnectionInfo | undefined> {
+    return Promise.resolve(undefined);
   }
 
-  static async save({
-    connectionInfo,
-    signal,
-  }: {
-    signal?: AbortSignal;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  save(options: {
     connectionInfo: ConnectionInfo;
-  }): Promise<void> {
-    throwIfAborted(signal);
-    try {
-      // While saving connections, we also save `_id` property
-      // in order to support the downgrade of Compass to a version
-      // where we use storage-mixin. storage-mixin uses this prop
-      // to map keytar credentials to the stored connection.
-
-      const { secrets, connectionInfo: connectionInfoWithoutSecrets } =
-        extractSecrets(connectionInfo);
-
-      let connectionSecrets: string | undefined = undefined;
-      try {
-        connectionSecrets = this.encryptSecrets(secrets);
-      } catch (e) {
-        log.error(
-          mongoLogId(1_001_000_202),
-          'Connection Storage',
-          'Failed to encrypt secrets',
-          { message: (e as Error).message }
-        );
-      }
-      await this.userData.write(connectionInfo.id, {
-        _id: connectionInfo.id,
-        connectionInfo: connectionInfoWithoutSecrets,
-        connectionSecrets,
-        version: this.version,
-      });
-      await this.afterConnectionHasBeenSaved(connectionInfo);
-    } catch (err) {
-      log.error(
-        mongoLogId(1_001_000_103),
-        'Connection Storage',
-        'Failed to save connection',
-        { message: (err as Error).message }
-      );
-
-      throw err;
-    }
-  }
-
-  static async delete({
-    id,
-    signal,
-  }: {
-    id?: string;
     signal?: AbortSignal;
   }): Promise<void> {
-    throwIfAborted(signal);
-    if (!id) {
-      return;
-    }
-
-    try {
-      await this.userData.delete(id);
-    } catch (err) {
-      log.error(
-        mongoLogId(1_001_000_104),
-        'Connection Storage',
-        'Failed to delete connection',
-        { message: (err as Error).message }
-      );
-
-      throw err;
-    }
+    return Promise.resolve();
   }
 
-  static async load({
-    id,
-    signal,
-  }: {
-    id?: string;
-    signal?: AbortSignal;
-  }): Promise<ConnectionInfo | undefined> {
-    throwIfAborted(signal);
-    if (!id) {
-      return undefined;
-    }
-    const connections = await this.loadAll();
-    return connections.find((connection) => id === connection.id);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
If you're not using them, you don't need to define them (and can remove the eslint-disable): a method with no arguments is correctly implementing a method with some arguments